src/stratigraphy/groundwater/groundwater_extraction.py

```diff
@@ -219,6 +232,12 @@ def get_groundwater_info_from_lines(self, lines: list[TextLine], page: int) -> G
                     if extracted_date_str:
                         text = text.replace(extracted_date_str, "").strip()
                         date = extracted_date
+                        matched_lines_rect.append(line.rect)
+                else:
+                    # in case several dates are present, we skip the other dates
```

Comment:
Clearer description:
"If a second date is present in the lines around the groundwater key, then we skip this line, instead of potentially falsely extracting a depth value from the date."

Reply:
Thank you!